charts/app-config/templates/env-configmap.yaml

```diff
@@ -16,6 +16,9 @@ data:
   GOVUK_APP_DOMAIN_EXTERNAL: {{ .Values.publishingDomainSuffix }}
   GOVUK_ASSET_ROOT: https://{{ .Values.assetsDomain }}
   GOVUK_CSP_REPORT_URI: {{ .Values.cspReportURI | quote }}
+  {{- if .Values.disableSearchAutocomplete }}
+  GOVUK_DISABLE_SEARCH_AUTOCOMPLETE: "1"
```

Comment:
It doesn't actually matter as we expect applications to check just for the presence of the env var. I went for "1" rather than "true" as I didn't want to create an impression that "false" would override it - as any env var value will be an indication of disabling.